lib/utils/isStandardSyntaxColorFunction.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const isStandardSyntaxFunction = require('./isStandardSyntaxFunction');
+
+/**
+ * Check whether a function is standard syntax color function
+ *
+ * @param {import('postcss-value-parser').FunctionNode} node
+ * @returns {boolean}
+ */
+module.exports = function isStandardSyntaxColorFunction(node) {
+	if (!isStandardSyntaxFunction(node)) return false;
+
+	// scss rgba() function can accept a hex as the first param
+	for (const fnNode of node.nodes) {
+		if (fnNode.type === 'word' && fnNode.value.startsWith('#')) return false;
```

Comment:
Scss users probably use a color variable like `$black` in the function, then it should still return false.

Reply:
@kamechb See https://github.com/stylelint/stylelint/issues/5671.

That issue is labelled as ready to implement. Please consider [contributing](https://stylelint.io/contributing) if you have time.

There are [steps on how to fix a bug in a rule](https://stylelint.io/developer-guide/rules#fix-a-bug-in-a-rule) in the Developer guide.